.drone/drone.jsonnet

```diff
@@ -498,7 +498,7 @@ local manifest_ecr(apps, archs) = pipeline('manifest-ecr') {
 
 [
   pipeline('loki-build-image') {
-    local build_image_tag = '0.33.1-golangci.1.51.2',
+    local build_image_tag = '0.33.1',
```

Comment:
I was under the impression from another discussion that this `golangci.1.51.2'` was to be added. Asking just to make sure, is removing intentional?

Reply:
Yes, that change is only needed on the 2.8.x branch